### DISCUSSION TITLE
Implement set playtime for scrobbling

### DISCUSSION THREAD

**Discussion by justsurfinhere** [web-scrobbler/web-scrobbler#3447] [Ideas]
Hey, I wondered if we could get an option to set a playtime percentage for scrobbling? Right now too many tracks get scrobbled when I actually skim over them.

Greets

**Comment by hornbuckle** [comment]
I believe this is already in place. If a track duration is available, it will scrobble at 50%, otherwise it scrobbles after 30 seconds*.
*This time seems to vary.
